src/script/standard.h

```diff
@@ -32,12 +32,6 @@ class CScriptID : public BaseHash<uint160>
     explicit CScriptID(const ScriptHash& in);
 };
 
-/**
- * Default setting for -datacarriersize. 80 bytes of data, +1 for OP_RETURN,
- * +2 for the pushdata opcodes.
- */
-static const unsigned int MAX_OP_RETURN_RELAY = 83;
```

Comment:
Dropping a constant but then hardcoding it all over the place with no historical context is not a good idea.

Reply:
Can you please explain why ?

Reply:
Because future developers won't (easily) know where the number comes from.

Reply:
> Dropping a constant but then hardcoding it all over the place

Where have I hardcoded it? I deliberately used a different constant in some of the updated tests to make clear that after this change, it's just a number. In the others - just 4 lines - I can of course just add a comment in those few lines explaining what the number is.

Reply:
In transaction_tests it's either the same number of +-1 which is confusing (can be solved with a comment), but I see in other places you made it 100 which is indeed more clear.

Reply:
I reworked those unit tests with better comments and clearer constants, and expanded the test coverage a bit.